dataset/src/jvmMain/kotlin/org/jetbrains/kotlinx/dl/dataset/OnHeapDataset.kt

```diff
@@ -28,16 +28,20 @@ import kotlin.streams.toList
 /**
  * Basic class to handle features [x] and labels [y].
  *
- * It loads the whole data from disk to the Heap Memory.
+ * It loads the whole data from the disk to the Heap Memory.
+ *
+ * NOTE: Labels [y] should have shape <number of rows; number of labels>
```

Comment:
This note seems to be incorrect and the number of labels is equal to the number of images